README.md

```diff
@@ -100,50 +248,137 @@ This will keep the global `editor.formatOnSave` for other languages you use, and
+ "terraform-ls.ignoreDirectoryNames": [
+   "folder1",
+   "folder2"
+ ]
+```
 
-In addition, this new version brings the syntax highlighting up to date with all HCL2 features, as needed for Terraform 0.12 and above.
+### Terraform command options
 
-**Configuration Changes** Please note that in 2.x, the configuration differs from 1.4.0, if you are having issues with the Language Server starting, you can reset the configuration to the following:
+You can configure the path to the Terraform binary used by terraform-ls to perform operations inside the editor by configuring this setting:
 
 ```json
-{
-  "terraform.languageServer": {
-    "external": true,
-    "args": [
-      "serve"
-    ]
-  }
-}
+"terraform-ls.terraformExecPath": "C:/some/folder/path"
 ```
 
-See the [CHANGELOG](https://github.com/hashicorp/vscode-terraform/blob/main/CHANGELOG.md) for more information.
+You can override the Terraform execution timeout by configuring this setting:
 
-## Terraform 0.11
+```json
+"terraform-ls.terraformExecTimeout": "30"
+```
 
-If you are using a Terraform version prior to 0.12.0, you can install the pre-transfer version of this extension manually by [following the instructions in the wiki](https://github.com/hashicorp/vscode-terraform/wiki/Install-a-Pre-transfer-Version).
+You can set the path Terraform logs (TF_LOG_PATH) by configuring this setting:
+
+```json
+"terraform-ls.terraformLogFilePath": "C:/some/folder/path/log-{{.varName}}.log"
+```
+
+Supports variables (e.g. Timestamp, Pid, Ppid) via Go template syntax `{{.VarName}}`
+        
+
+### Telemetry
+
+We use telemetry to send error reports to our team, so we can respond more effectively. You can configure VS Code to send all telemetry, just crash telemetry, just errors or turn it off entirely by [configuring](https://code.visualstudio.com/docs/getstarted/telemetry#_disable-telemetry-reporting) `"telemetry.telemetryLevel"` to your desired value. You can also [monitor what's being sent](https://code.visualstudio.com/docs/getstarted/telemetry#_output-channel-for-telemetry-events) in your logs.
 
 ## Known Issues
 
-* Completion inside incomplete blocks, such as `resource "here` (without the closing quote and braces) is not supported. You can complete the 1st level blocks though and that will automatically trigger subsequent completion for e.g. resource types. See [relevant issue](https://github.com/hashicorp/terraform-ls/issues/57) for more.
-* A number of different folder configurations (specifically when your root module is not a parent to any submodules) are not yet supported. ([info](https://github.com/hashicorp/terraform-ls/issues/32#issuecomment-649707345))
+- If there are validation errors in the open document, intellisense does not provide inline completions. Run `Terraform: validate` and fix validation errors, then reload the document and intellisense will work again. This is being worked on in [terraform-vscode#123](https://github.com/hashicorp/terraform-vscode/issues/123).
```

Comment:
I think the URL will need replacing here.

Reply:
It was a placeholder until I could find the issue to reference here. Updated